packages/core/src/rules/oas3/remove-unused-components.ts

```diff
@@ -31,7 +31,11 @@ export const RemoveUnusedComponents: Oas3Rule = () => {
         ) {
           const resolvedRef = resolve(ref);
           if (!resolvedRef.location) return;
-          components.set(resolvedRef.location.absolutePointer, {
+          const outerParts = resolvedRef.location.absolutePointer.split('#', 2);
+          const parts = outerParts[1].split('/').slice(0, 4)
```

Comment:
Also, I would rename variables to clarify the intent to extract only the #/components/schemas/<NAME> part.

Reply:
@RomanHotsiy I've renamed the variables (and moved the `join` up a line).

Question - was this comment meant to be in addition to another comment? Starting with "Also" suggests there's some other feedback but I'm not seeing anything else.

Reply:
ah, yes, sorry, it was initially but then I edited it out but forgot to remove "also" 😄